recipe/meta.yaml

```diff
@@ -13,32 +15,79 @@ source:
   sha256: 23793c8645480148e9559df96b386b780f92194c80120acce79fcdaae0d81f45
 
 build:
-  number: 0
-  script: python build/build.py {{ args }}                                          # [unix]
-  script: python build/build.py --make-tool=mingw32-make {{ args }}                 # [win]
+  number: 1
   script_env:
     - OS    # [win]
-  features:
-    - vc9   # [win and py27]
-    - vc14  # [win and py>=35]
 
 requirements:
   build:
     - toolchain
```

Comment:
Also we can drop this now that we are using `compiler` below.

Reply:
done. BTW, dropping features results in too similar names for non-python packages where only hash string distinguishes between vc9 and vc14 versions.. Is it desired behavior, @jakirkham ?